src/main/java/edu/ucla/library/prl/harvester/services/HarvestScheduleStoreServiceImpl.java

```diff
@@ -75,25 +83,36 @@ public class HarvestScheduleStoreServiceImpl implements HarvestScheduleStoreServ
     /**
      * The update query for an institution.
      */
-    private static final String UPDATE_INST = "UPDATE public.institutions SET name=$1, description=$2," +
-            " location=$3, email=$4, phone=$5, webContact=$6, website=$7 WHERE id = $8";
+    private static final String UPDATE_INST = """
+        UPDATE public.institutions SET name=$1, description=$2, location=$3,
+        email=$4, phone=$5, webContact=$6, website=$7 WHERE id = $8
+        """;
 
     /**
      * The select-one query for jobs.
      */
-    private static final String GET_JOB = "SELECT * FROM public.harvestjobs WHERE id = $1";
+    private static final String GET_JOB = """
+        SELECT institutionId AS "institutionID", repositoryBaseUrl AS "repositoryBaseURL",
+        metadataPrefix AS "metadataPrefix", sets, lastSuccessfulRun AS "lastSuccessfulRun",
+        scheduleCronExpression AS "scheduleCronExpression" FROM public.harvestjobs WHERE id = $1
+        """;
 
     /**
      * The insert query for jobs.
      */
-    private static final String ADD_JOB =
-            "INSERT INTO public.harvestjobs(institutionID, repositoryBaseURL, metadataPrefix, sets," +
-                    " lastSuccessfulRun, scheduleCronExpression) VALUES($1, $2, $3, $4, $5, $6) RETURNING id";
+    private static final String ADD_JOB = """
+        INSERT INTO public.harvestjobs(institutionId, repositoryBaseUrl, metadataPrefix, sets,
+        lastSuccessfulRun, scheduleCronExpression) VALUES($1, $2, $3, $4, $5, $6) RETURNING id
+        """;
 
     /**
-     * The select query for all jobs.
+     * The select query for all jobs. N.b. awkward format to satisfy checkstyle rules
```

Comment:
Assuming the use of Heredocs changes this:

```suggestion
     * The select query for all jobs.
```

Reply:
Removed obsolete comment